js/app/modules/contentGroup/contentGroup.js

```diff
@@ -13,6 +13,7 @@ const Overflow = imports.app.modules.arrangement.overflow;
 const Utils = imports.app.utils;
 
 const BATCH_SIZE = 10;
+const EXTRA_CLIP = 40;
```

Comment:
This is a bit of a magic number; it would be best to leave a comment saying why it's 40px.